### PR TITLE
Adding motion planning and trajectory optimization details to the research page

### PR DESCRIPTION
**Pull request by nataliya-dev**
- Adding motion planning and trajectory optimization details to the research page
- Moving Conor from undergrad to alumni
- Adding Yaashia as a collaborator
- Adding a link to my web page

The is the first part of the updates. Caleb is going to build on to these updates in his subsequent PRs.

### CODE REVIEW

File: _posts/people/2023-10-24-nataliya.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Nataliya Nechyporenko
+description: PhD Student
+layout: redirect
+redirect_to: https://nataliya.dev
+permalink: nataliya.html
+sitemap: false
+---
```
Comment (alecive) [inline _posts/people/2023-10-24-nataliya.md:8]: I didn't know it was possible, and that's cool. However, if you don't have it in the sitemap, what's the purpose of having this page in the first place ?

File: _posts/research/subteams/2021-03-16-control-artificial-skin.md [hunk 2]
```diff
@@ -65,6 +65,28 @@ Future work in this area will focus on gentle continuous contact interactions, l
 
 {% include image.html url="research/roboskin/sensor_units.jpg" max-width="40%" description="Anticipating contact with whole-body sensing." %}
 
+
+
+## 1.3 Motion planning and trajectory optimization for contact-rich manipulation
```
Comment (alecive) [inline _posts/research/subteams/2021-03-16-control-artificial-skin.md:70]: I think we should place these sections in reverse chronological order. Therefore, I would put this one as section 1.1 as it's the most recent.

Comment (alecive) [reply _posts/research/subteams/2021-03-16-control-artificial-skin.md:70]: Looking at how nice the below is, I would put it first :-)

Comment (nataliya-dev) [reply _posts/research/subteams/2021-03-16-control-artificial-skin.md]: Updated as section 1.1

File: _posts/people/2023-10-24-nataliya.md (hunk 1 above)
Comment (nataliya-dev) [reply _posts/people/2023-10-24-nataliya.md:8]: Also updated, the goal is to redirect my image in "people" to my website.

Comment (alecive) [reply _posts/people/2023-10-24-nataliya.md:8]: Yeah sorry what I meant was that I didn't know it was possible to have a redirect just like that , but if we do we need to have it in the sitemap.

**Review by alecive — APPROVED** [review]

File: _posts/people/2023-10-24-nataliya.md (hunk 1 above)
Comment (nataliya-dev) [reply _posts/people/2023-10-24-nataliya.md:8]: Yep, I added sitemap: True. I believe that's the change you requested. This conversation doesn't show that change in my last push, but it's in the FilesChanged.

**Comment by alecive** [comment]
Ref #201

**Pull request merged**